karapuzio/assignment3/WordSearch.java

```diff
@@ -0,0 +1,141 @@
+package edu.codeU.assignment3;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+
+/**
+ * The class to find all words in the grid.
+ */
+public class WordSearch {
+    /**
+     * The number of rows in grid.
+     */
+    private int rows;
+    /**
+     * The number of columns in grid;
+     */
+    private int columns;
+    /**
+     * The grid in which we look words.
+     */
+    private char[][] grid;
+    /**
+     * The List of Words in dictionary.
+     */
+    private List<String> dict;
```

Comment:
I understand that you decided not implement the class Dictionary? As I mentioned earlier, I am not requiring this but recommending as an exercise. So I just wanted to double check with you

Reply:
If you don't mind, I'm going to implement it these weekends.